### PR TITLE
chore: misc PHPCS fixes

### PR DESCRIPTION
**Pull request by justlevine**
This PR addresses 17 failing PHPCS smells.

While production code has been updated, *no user-facing / api functionality has been changed*. 

- [x] Yes I signed the contribution agreement.

## Details
- Fixed the usage of `wp-graphql` as a text domain when it should be `wp-graphql-content-blocks`
- Fixed a handful of issues regarding string translation (improper / missing use of `sprintf`, non-escaped strings, unnecessary interpolation).
- Removed the unused params passed from the block attribute field resolver to our anonymous callback.
- Fixed a yoda conditional.

### CODE REVIEW

**Comment by theodesp** [comment]
@justlevine Thanks. Could you add a changeset as well? Just run npm run changeset and use patch version.

**Comment by justlevine** [comment]
@theodesp do you want this as a single changeset or 4? (these are individual types of code changes, but none - individually or collectively - affect a user's usage)

**Review by theodesp — DISMISSED** [review]

**Comment by theodesp** [comment]
@justlevine one changeset for each PR so we don't miss anything. Thanks.

File: includes/WPGraphQLContentBlocks.php [hunk 1]
```diff
@@ -117,9 +117,9 @@ function () {
 
 						echo sprintf(
 							'<div class="notice notice-error">' .
-							'<p>%s</p>' .
+								'<p>%s</p>' .
 							'</div>',
-							__( 'WPGraphQL Content Blocks appears to have been installed without it\'s dependencies. If you meant to download the source code, you can run `composer install` to install dependencies. If you are looking for the production version of the plugin, you can download it from the <a target="_blank" href="https://github.com/wpengine/wp-graphql-content-blocks/releases">GitHub Releases tab.</a>', 'wp-graphql' )
+							esc_html__( 'WPGraphQL Content Blocks appears to have been installed without it\'s dependencies. If you meant to download the source code, you can run `composer install` to install dependencies. If you are looking for the production version of the plugin, you can download it from the <a target="_blank" href="https://github.com/wpengine/wp-graphql-content-blocks/releases">GitHub Releases tab.</a>', 'wp-graphql-content-blocks' )
```
Comment (mindctrl) [inline includes/WPGraphQLContentBlocks.php:122]: I'm just eyeballing this, but won't `esc_html__` cause the link to not be clickable?

Comment (mindctrl) [reply includes/WPGraphQLContentBlocks.php:122]: re: broken link..

image

Comment (justlevine) [reply includes/WPGraphQLContentBlocks.php:122]: good catch! that should probably changed to a `wp_kses_*`. 
I had to switch back to client work for a bit, but Ive got `allow edits from maintainers` on if you don't want to way

**Review by mindctrl — APPROVED** [review]

**Review by theodesp — APPROVED** [review]

**Pull request merged**